e2e_test/database/test.slt

```diff
@@ -0,0 +1,3 @@
+query T
+show schemas;
```

Comment:
database/  Did you accidentally push the irrelevant files in the wrong PR?

Reply:
Oh, let me revert this